### PR TITLE
Add swagger settings & configs

### PR DESCRIPTION
**Pull request by ckertam**
Swagger UI added we can see it using below endpoint in localhost
[Local Swagger](http://localhost:8000/user/api/swagger)
[Dev Swagger](http://34.72.72.115:8000/user/api/swagger)
[Prod Swagger](http://35.194.29.12:8000/user/api/swagger)

### CODE REVIEW

File: backend/user/urls.py [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
+from rest_framework import permissions
+from rest_framework_swagger.views import get_swagger_view
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Django Sample Application API",
+        default_version='v1',
+        description="Welcome to the Django Sample Application API documentation",
```
Comment (oguzsenna) [inline backend/user/urls.py:14]: Let's change this to be like 'SWE574 G1 Memories Application API Documentation'.

File: backend/user/urls.py [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
 from django.conf import settings
 from django.conf.urls.static import static
+from rest_framework import permissions
+from rest_framework_swagger.views import get_swagger_view
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Django Sample Application API",
```
Comment (oguzsenna) [inline backend/user/urls.py:12]: Let's change this to be like 'SWE574 G1 Memories Application API'.

Comment (ckertam) [reply backend/user/urls.py]: I changed the Title! 🥂

File: backend/user/urls.py (hunk 1 above)
Comment (ckertam) [reply backend/user/urls.py]: I changed the Title! 🦤

**Review by oguzsenna — APPROVED** [review]
More Passion , More energy 🍒

**Pull request merged**